src/index.js

```diff
@@ -1,9 +1,26 @@
 'use strict';
 
 const NewsEngine = require('./news-engine');
+const config = require('../config');
+const fs =  require(`fs`);
+const uuid = require(`uuid`);
+const svg2png = require('svg2png');
 
 const numExamples = process.argv[2] || 30;
 const headlines = NewsEngine.generateHeadlines(numExamples);
 headlines.forEach(headline => {
 	console.log("\n * " + headline.text);
+
+	if(headline.tags && headline.tags.svg && headline.tags.svg.svgString) {
+		const outputfile = uuid() + '.png';
+		const outputPath = config.paths.tempDirectory + '/' + outputfile;
+
+		svg2png(new Buffer(headline.tags.svg.svgString))
```

Comment:
I'll mess around with some config for this later. I'll probably set up a specific /tmp dir on the server for this app to use for everything.

BTW for gggolem I just let card images pile up in a folder forever (each card gets a deterministic filename and 
it uses this as a cache for gatherer results)

That obviously won't work for this use case, but the vps has plenty of space for now.